src/main/java/edu/hm/hafner/analysis/parser/AbstractMavenLogParser.java

```diff
@@ -12,17 +15,52 @@
 
 public class AbstractMavenLogParser extends MavenConsoleParser {
 
+
+
+    private static final String WARNING = "WARNING";
+    private static final String ERROR = "ERROR";
+
+    /**
+     * Regular expression to parse the start of maven plugin in console.
+     */
+    protected static final Pattern MAVEN_PLUGIN_START = Pattern.compile(
+            "\\[INFO\\] --- (?<id>\\S+):(?<version>\\S+):(?<goal>\\S+)\\s.*");
+
+    private static final Pattern MAVEN_MODULE_START = Pattern.compile(
+            "-+< (?<id>\\S+) >-+"
+    );
+
     private String goal = StringUtils.EMPTY;
+    private String module = StringUtils.EMPTY;
+
 
-    public boolean isLineInteresting(final String line) {
-        return super.isLineInteresting(line);
+    @Override
+    protected boolean isLineInteresting(final String line) {
+        Matcher goalMatcher = MAVEN_PLUGIN_START.matcher(line);
+
+        if (goalMatcher.find()) {
+            goal = String.format("%s:%s", goalMatcher.group("id"), goalMatcher.group("goal"));
+        }
+
+        if(!goal.isEmpty()){
```

Comment:
Hey @uhafner 


Could you confirm if I have shifted the correct code  from `MavnConsoleParser`.

Because though the `AntJavacParserTest` succeeds, other 10 test casess fail :(